src/client/app/store/services/app.dispatchers.ts

```diff
@@ -1,12 +0,0 @@
-import { Injectable } from '@angular/core';
```

Comment:
Great that we are removing this and the other files like this (dispatchers and selectors). Let's make sure this is added to the documentation so folks know how to extend dispatchers and selectors when desired.